docs/04-api-reference/front-commerce-remix/defineRemixExtension.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: "defineRemixExtension"
+sidebar_position: 4
+description:
+  "defineRemixExtension allow you to extend the features of Front-Commerce for a
+  Remix application."
+---
+
+<p>
+  Extensions are registered in the application using the
+  front-commerce.config.ts file. Read [Register an
+  extension](/docs/remixed/guides/register-an-extension) to learn more.
+</p>
```

Comment:
**Issue:** The `Register and extension` link is broken:

![image](https://github.com/front-commerce/developers.front-commerce.com/assets/4162177/aa41e467-a9be-4ce9-9f98-ce427043ab52)